File: JAVA/Ruster exercise/com/Ruster/Pension.java
/**
 *
 * @author JUAN
 */

package com.Ruster;

public class Pension {
    private double calculatePension;

    public Pension(double salary) {
        this.calculatePension = salary * 0.04; // 4%
    }

    public double getCalculatePension() {
        return this.calculatePension;
    }
}

File: PHP/Ruster exercise/Ruster/Pension.php
<?php
class Pension extends Person {
    private float $calculatePension;

    public function __construct(float $salary) {
        $this->salary = $salary;
        $this->calculatePension = $this->salary * 0.04;
    }

    public function getCalculatePension(): float {
        return $this->calculatePension;
    }

    public function getDescription(): string {}
    public function getNetSalary(): float {}
}


